LayoutTests/fast/forms/meter-appearance-on-pseudo-element.html

```diff
@@ -0,0 +1,18 @@
+<style>
+    dialog::backdrop {
+        content: "";
+        display: inline-block;
+        appearance: meter;
+        width: 16px;
+        height: 100px;
```

Comment:
nit: note that this size was chosen for `slider-vertical`, although it doesn't matter much

Reply:
Removed.